### PR TITLE
Bugfix/sp 525 fix author section design in mobile screen

### PR DESCRIPTION
**Pull request by BahriRanim**

### CODE REVIEW

File: src/components/AuthorBlock/AuthorBlock.module.scss [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
   }
 }
 .blockWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-right: -15px;
+  margin-left: -15px;
   @media (min-width: 768px) {
     display: flex !important;
```
Comment (bouthouri) [inline src/components/AuthorBlock/AuthorBlock.module.scss:21]: you don't need this

**Review by bouthouri — CHANGES_REQUESTED** [review]
You did not optimize the CSS as I told you and I can see !important also.
Fix this and let me know.

File: src/components/AuthorBlock/AuthorBlock.module.scss (hunk 1 above)
Comment (BahriRanim) [reply src/components/AuthorBlock/AuthorBlock.module.scss]: Done

**Review by bouthouri — APPROVED** [review]

**Comment by bouthouri** [comment]
Task linked: [SP-525 Fix author section design in mobile screen](https://app.clickup.com/t/2427294/SP-525)

**Pull request merged**